packages/style/components/filters/CheckboxFilter.module.scss

```diff
@@ -80,8 +79,6 @@
         color: $checkbox-filter-actions-separator-links;
         text-transform: capitalize;
         padding: 0;
-        font-size: 1.2rem;
-        line-height: 2rem;
```

Comment:
Why do we remove this ? for CQDG I cant say that all my labels in checkbox are 1.2rem because I have other checkboxes that uses the 1.4rem

Reply:
specify ant level css in you ant design theme

Reply:
So ant has to know in which context he reside ?